subprojects/ced.wrap

```diff
@@ -0,0 +1,5 @@
+[wrap-file]
+directory=compact_enc_det-meson
+source_url=https://github.com/performous/compact_enc_det/archive/refs/heads/meson.zip
```

Comment:
One last thing i'd merge the `meson` branch within ced to master and use that here

Reply:
Agreed. Makes sense. Could you do it, please? Today's looking like it's gonna be a very busy day.

Reply:
Aight